src/renderer/components/player/ImagePlayer.tsx

```diff
@@ -7,6 +7,7 @@ import fs from "fs";
 import gifInfo from 'gif-info';
 import ChildCallbackHack from './ChildCallbackHack';
 import urlToPath from '../../urlToPath';
+import { isNull, isNullOrUndefined } from 'util';
```

Comment:
These aren't used, please remove them

Reply:
- Spurious unused import has now been removed (not sure where that artefact came from)

- Additions post original pull request
I have added a Simple Color Picker (SimpleColorPicker.tsx)  - I am not sure if the approach I took to the isVisible property and its use in the render method is the approved way of doing things - but the effect in the Scene Details is to only display the SimpleColorPicker when the user chooses a fixed color for the background - when the choice is a blurred image the SimpleColorPicker is removed.   There may be cleaner methods of achieving this - let me known and I'll have a go at remediation.